### DISCUSSION TITLE
Is this a problem with wiki or this version of termux?

### DISCUSSION THREAD

**Discussion by Djspac3** [termux-play-store/termux-issues#131] [Q&A] [answered]
I noticed that the config file for termux is under ~/.config/termux but the wiki says ~/.termux/
is that like just not updated wiki or something I have to figure out?

**Comment by 09680968** [comment]
cheat pubgji

**Reply by Djspac3** [reply]
what?
the hell does that mean

**Reply by matinkunduzi511-star** [reply]
> cheat pubgji 
> Send WhatsApp text code, thank you.I want the WhatsApp report text code, please.

**Reply by matinkunduzi511-star** [reply]
Send me the WhatsApp report text code.

**Comment by Djspac3** [comment, marked as answer]
realized its just both either one works
ruddendency i guess

**Reply by matinkunduzi511-star** [reply]
سلام علیکم وقتتون بخیر لطفاً از شما یه خواهش داشتم 0785569381این اکانت واتساپ من را رفع کنید محدود شده است

**Comment by matinkunduzi511-star** [comment]
>
Send WhatsApp report text code, thank you, my love.
